backend/spec/controllers/spree/admin/orders_controller_spec.rb

```diff
@@ -296,19 +296,12 @@
     end
 
     context "#open_adjustments" do
-      let(:closed) { double('closed_adjustments') }
-      let(:closed_adjustment) { double }
-
-      before do
-        allow(adjustments).to receive(:where).and_return(closed)
-        allow(closed).to receive(:each).and_return([])
-      end
+      let(:order) { create(:order) }
+      let!(:closed_adjustment) { create(:adjustment, finalized: true, adjustable: order, order: order) }
```

Comment:
Ha, oops. I misread that. I thought you were asking for a `finalized_adjustment` factory.  Fixing now.